app/frontend/mocks/projectHandlers.ts

```diff
@@ -11,4 +19,38 @@ export const projectHandlers = [
     )
     return result
   }),
+  mockProjectMembershipDeleteMutation((request, response, context) => {
+    const indexDeleteProject = assignedProjects.findIndex((ap) => ap.id === request.variables.projectID)
+
+    assignedProjects = assignedProjects.slice(indexDeleteProject + 1)
```

Comment:
This will not work for `indexDeleteProject` > 0.

Maybe we can use `.filter()` method?